### ISSUE TITLE
Logo getting cutoff on smaller screens on Citizen Engagement page

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4226]
### Overview
On the Citizen Engagement page's Organizations We Work With section, we need to fix the logo so it doesn't get cutoff in tablet and mobile views.

### Details
- Screenshot of problem:
  <details>
    <summary>Click here to see a gif of the logo being cutoff as it the view becomes smaller</summary>
    <img src="https://user-images.githubusercontent.com/31293603/226738334-6ab39976-e85b-43db-9ad4-823f5ce99c6d.gif" alt="">
  </details> 

- Here are screenshots of the Figma Design of the Organizations We Work With section of the Citizen Engagement page. For dimensions and other specifications, please inspect the Figma Design for the [Citizen Engagement design in the red box titled Newest Version](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3625-0&t=xpc1QvW41BRJ0jNW-0) to see how it should look for desktop and mobile views. 

  <details>
    <summary>Click here to see screenshots of the Organizations We Work With section of the Citizen Engagement page in mobile and desktop views</summary>
  
    <b>Desktop</b>
    <img width="708" alt="" src="https://user-images.githubusercontent.com/31293603/229689465-344865e2-f858-4e6f-9e4f-dac7c8358ba9.png">
  
    <b>Mobile</b>
    <img width="211" alt="" src="https://user-images.githubusercontent.com/31293603/229689516-1eaf8dc1-1149-45f9-98f9-d6a288ae1dcc.png">
  
  </details>

### Action Items
- [ ] On the Citizen Engagement page's Organizations We Work With section, fix the logo so it resizes as the screen width gets smaller and smaller. (See the Resources/Instructions' Files section for some files to get you started.) 
- [x] Make sure nothing on the website is broken in mobile, tablet, and desktop views

#### Merge Team, Technical Lead, or Product Manager
- [ ] Once the pull request associated with this issue is merged, go to issue #3328 and check if all the issues listed under the `Dependency` section are closed. If they are all closed, move that issue (#3328) into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it. 

### Resources/Instructions
- Citizen Engagement page: https://www.hackforla.org/citizen-engagement

#### Files to get you started
- [pages/citizen-engagement.html](https://github.com/hackforla/website/blob/gh-pages/pages/citizen-engagement.html)
- [_sass/components/_citizen_engagement.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/components/_citizen_engagement.scss)

#### Additional resources
- [Liquid template language](https://shopify.github.io/liquid/)
- [SCSS](https://sass-lang.com/documentation/)
- See the Figma Design [Citizen Engagement design in the red box titled Newest Version](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3625-0&t=xpc1QvW41BRJ0jNW-0)

**Comment by mademarc** [comment]
Progress: Gonna work on this issue and have it done by end of the month.  
Blockers: no blockers yet   
Availability: 3/28/2023 
ETA: 3/31/2023

**Comment by mademarc** [comment]
Availability: 4/5/2023
ETA: 4/7/2023

**Comment by ExperimentsInHonesty** [comment]
@mademarc I am taking this issue back.  We are short of issues and it hurts team velocity when people assign issues before they are ready to work on them.

Please self assign another issue from the backlog when you return next week.

**Comment by Adaakal** [comment]
i. Availability: Monday 3/27 at 1:30pm - 3:30pm; Thursday 3/30 at 12:30pm - 3:30pm
ii. ETA: Sunday 4/2

**Comment by Adaakal** [comment]
**Progress: "What is the current status of your project? What have you completed and what is left to do?"**
I submitted the issue for review and received some notes. So I'm following @RyanGehris review and looking into how I can redo my issue. 
**Blockers: "Difficulties or errors encountered."**
I need to make sure my changes are in line with the current Figma design rules. 
**Availability: "How much time will you have this week to work on this issue?"**
Maybe one hour this weekend.
**ETA: "When do you expect this issue to be completed?"**
Hoping to have the PR re-submitted by EOD Monday, 4/3.

**Comment by Adaakal** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"
I have to unassign myself from this issue due to scheduling conflicts. 
Blockers: "Difficulties or errors encountered."
Schedule.
Availability: "How much time will you have this week to work on this issue?"
N/A
ETA: "When do you expect this issue to be completed?"
N/A

**Comment by StephenTheDev1001** [comment]
i. Availability: M-F 4pm-7pm
ii. ETA: Apr 7th

**Comment by ronaldpaek** [comment]
i. Availability: Fri-Sun 5 pm - 9 pm
ii. ETA: 04/10/23

**Comment by ronaldpaek** [comment]
@StephenTheDev1001 Oh, oops are you working on this task, I saw that it was still in the prioritized backlog

**Comment by StephenTheDev1001** [comment]
@ronaldpaek This one is intimidating. You should take this one.

**Comment by ronaldpaek** [comment]
@StephenTheDev1001 sure, thanks

**Comment by ronaldpaek** [comment]
i. Availability: Fri-Sun 5 pm - 8 pm
ii. ETA: 4/22/23 

I was not able to work on this because I am in the process of moving and looking for a new place to live. Sorry, I am going to work on this task on Friday.

**Comment by roslynwythe** [comment]
Hi @ronaldpaek please leave a comment outlining your progress, availability and ETA, and describe any problems or blockers you might have, then remove the "2 weeks inactive" label.   If you are not planning to work on this issue, please let us know.  Thanks!

**Comment by ExperimentsInHonesty** [comment]
@ronaldpaek I am removing you from this issue.  You are always welcome to take time off, but you should not assign yourself to an issue until you are available to work on it.  Otherwise you are keeping other people who are looking for issues, from working on it.

**Comment by jch1013** [comment]
Availability: Any afternoon
ETA: May 23

**Comment by jch1013** [comment]
Availability: Any afternoon
ETA: June 1
Blockers: Updated my computer software this week and it caused an issue with my git, have been working to resolve my git issue before I can continue working on this issue.

**Comment by jch1013** [comment]
Progress: Understanding of the issue and possible plan for a temporary fix
Blockers: Unsure if the current CSS styling will work as more logos are added to this area; need to get advice on how best to create a long term fix. Plan to bring issue up at next meeting
Availability: after 3pm on weekdays
ETA: June 15, possibly sooner or later depending on what we decide at the meeting
